### ISSUE TITLE
New PR for https://github.com/julianpeeters/sbt-avrohugger/pull/25

### ISSUE THREAD

**Issue by julianpeeters** [julianpeeters/sbt-avrohugger#32]
Seeing an unexpected error in the scripted tests after merging conflicting #25 and #30, so I reverted #25 and will try #30 solo first. This means #25 must be resubmitted or otherwise dealt with.

**Comment by mariussoutier** [comment]
Yes I will resubmit #25 after merging manually.

**Comment by mariussoutier** [comment]
So you added the scripted tests back in? Can you commit them? Then I could merge and fix it all.

**Comment by julianpeeters** [comment]
No, haven't had time to add them back in, or think about how ti organize them.

I moved some of the tests, like the OverrideSettings sbt-test project, into avrohugger because I needed to test some of avrohugger's contribution, and felt that keeping them in sbt-avrohugger as well was duplication.

Now I'm of the opinion that a little duplication would be good if it serves to test separate functions, and mostly I was thinking of adding more tests, not moving tests back. So what is left to be done is to decide what tests would be good to have in sbt-avrohugger, regardless of duplication.
